### PR TITLE
Update user map for May 2016 #5538

### LINKED ISSUE #5538: Update user map for May 2016
Follow #5163 
List of institutes to add are:

```
Australia   University of Melbourne
Australia   University of South Australia
Australia   Wesley College
Australia   TAFE Coomera
Australia   Swinburne University
Australia   Suzanne Cory High School
Belgium Vrije Universiteit Brussel 
Belgium Virgo Sapiens
Brazil  Associação Educacional Dom Bosco 
Brazil  Instituto Federal do Espírito Santo
Brazil  UniFOA
Canada  Simon Fraser University
Canada  Collège Regina Assumpta
Canada  University of New Brunswick
Canada  Georgian College
Canada  University of Manitoba
Ghana   Ashesi Unuiversity
Malaysia    Universiti Malaysia Sarawak
Malaysia    Universiti Sains Malaysia
Malaysia    University of Nottingham Malaysia Campus
Mexico  UNID
Netherlands Vrije Universiteit Amsterdam
Netherlands Teylingen College Leeuwenhorst
New Zealand University of Otago
Scotland    The Glasgow School of Art
Singapore   Singapore Institute of Technology
Singapore   Ngee Ann Polytechnic
UK  University of East London
USA Creighton University
USA Saint Mary's College
USA San Diego State University
USA New Mexico State University
USA Hopkins North Junior High School
USA Washington State University
USA Texas A&M University
USA Dawson high school
USA University of Texas at Arlington
USA Miami University
USA Louis Pasteur Middle School 67
USA University of Texas at Austin
USA North Carolina State University
USA Morgan State University
USA University of Louisville
USA Lehigh University
USA Carnegie Mellon University
USA University of Arizona South
USA Squalicum Hight School
USA BYU Idaho Online
USA Miller Career & Technical Center 
USA University at Buffalo
USA Newport News Schools 
USA Penn State University
USA University of Arizona
USA Calvin College
USA Tennessee Tech University
USA Allen High School
USA University of Kentucky
USA California College San Diego
USA University of Texas at Arlington
USA Kenan-Flagler Business School
```

### PR DESCRIPTION
**Pull request by mariasam1**
Fixes #5538 

Added the colleges for June, and also updated date to "30 June 2016". There were some colleges in the post that were already in userMapData.json, so I didn't add those again.

### CODE REVIEW

File: src/main/webapp/js/userMapData.json [hunk 1]
```diff
@@ -234,16 +247,19 @@
...
 ["TU Delft","Netherlands"],
 ["University of Inholland","Netherlands"],
 ["University of Twente","Netherlands"],
 ["Utrecht University","Netherlands"],
+["Vrije Universiteit Amsterdam","Netherlands"],
 ["VU University","Netherlands"],
 ["Windesheim Honours College","Netherlands"],
 ["Unitec Institute of Technology","New Zealand"],
 ["University of Auckland","New Zealand"],
+["University of Otago","New Zealand”],
```
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: Different double quote after `New Zealand` here, use `"` instead of `”`

Comment (mariasam1) [inline src/main/webapp/js/userMapData.json]: Got it, fixed.

File: src/main/webapp/js/userMapData.json [hunk 2]
```diff
@@ -139,6 +148,7 @@
 ["Universität der Bundeswehr München","Germany"],
 ["University of Paderborn","Germany"],
 ["Wescale","Germany"],
+["Ashesi Unuiversity","Ghana"],
```
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: Maybe fix the typo here ie. `"Ashesi University"`

File: src/main/webapp/js/userMapData.json [hunk 3]
```diff
@@ -456,6 +479,7 @@
 ["Dalton State College","USA"],
 ["Dartmouth College","USA"],
 ["Davis Senior High School","USA"],
+["Dawson high school","USA"],
```
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: Capitalise this to keep it consistent with the other entries: `"Dawson High School"`

File: src/main/webapp/js/userMapData.json [hunk 4]
```diff
@@ -664,6 +697,7 @@
 ["Southwestern Law School","USA"],
 ["Spelman College","USA"],
 ["Spring Hill College","USA"],
+["Squalicum Hight School","USA"],
```
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: Fix the typo here: `"Squalicum High School"`

**Comment by chowyb** [comment]
A few minor changes to address, but looks good otherwise :+1:

**Comment by chowyb** [comment]
Okay the recent fix looks good, but you'll now need to merge the `master` branch into this branch, and fix the merge conflicts, as your previous PR might have conflicting changes with this one.

**Comment by damithc** [comment]
I updated the PR title to fix an error in the issue title. the branch name can stay the same although it doesn't match.

**Comment by chowyb** [comment]
Alright, @mariasam1 you'll have to change the date to 31 May as well then

**Comment by wkurniawan07** [comment]
@mariasam1 there is a merge conflict that you need to resolve.

**Comment by wkurniawan07** [comment]
@Mariasam1 any updates here?

**Comment by wkurniawan07** [comment]
Closing due to inactivity.

**Pull request closed without merging**